File: GeralViewController/web/js/RepDiagnostico.js
function gravarInfoDiagnostico() {
    var form = document.getElementById("formDiagnostico");

    var numAnimal = document.getElementById("numAnimal").value;
    var nomAnimal = document.getElementById("nomAnimal").value;

    var inseminadasCobertas = document.getElementById("inseminadasCobertas").value;
    var vazias = document.getElementById("vazias").value;
    var gestantes = document.getElementById("gestantes").value;
    var emCrescimento = document.getElementById("emCrescimento").value;

    var padrao = document.getElementById("padrao").value;
    var doadora = document.getElementById("doadora").value;
    var receptora = document.getElementById("receptora").value;
    var descarte = document.getElementById("descarte").value;

    var dataDiagnostico = document.getElementById("dataDiagnostico").value;
    var horaDiagnostico = document.getElementById("horaDiagnostico").value;

    var diagnostico = document.getElementById("diagnostico").value;
    var sexo = document.getElementById("sexo").value;

    var numDias = document.getElementById("numDias").value;
    var reprodutor = document.getElementById("reprodutor").value;
    var raca = document.getElementById("raca").value;

    var dataCio = document.getElementById("dataCio").value;
    var horaCio = document.getElementById("horaCio").value;

    var cio = document.getElementById("cio").value;
    var muco = document.getElementById("muco").value;
    var te = document.getElementById("te").value;

    var horaCio = document.getElementById("utero").value;
    var cio = document.getElementById("ovd").value;
    var muco = document.getElementById("ovE").value;
    var te = document.getElementById("base").value;

    if (numAnimal === "") {
        alert("O número do animal é obrigatório.");
        document.getElementById("numAnimal").focus();
        return false;
    } else if (nomAnimal === "") {
        alert("O nome do animal é obrigatório.");
        document.getElementById("nomAnimal").focus;
        return false;
    } else if (inseminadasCobertas === "") {
        alert("Campo obrigatório.");
        document.getElementById("inseminadasCobertas").focus;
        return false;
    } else if (vazias === "") {
        alert("Campo obrigatório.");
        document.getElementById("vazias").focus;
        return false;
    } else if (gestantes === "") {
        alert("Campo obrigatório.");
        document.getElementById("gestantes").focus;
        return false;
    } else if (emCrescimento === "") {
        alert("Campo obrigatório.");
        document.getElementById("emCrescimento").focus;
        return false;
    } else if (padrao === "") {
        alert("Campo obrigatório.");
        document.getElementById("padrao").focus;
        return false;
    } else if (doadora === "") {
        alert("Campo obrigatório.");
        document.getElementById("doadora").focus;
        return false;
    } else if (receptora === "") {
        alert("Campo obrigatório.");
        document.getElementById("receptora").focus;
        return false;
    } else if (descarte === "") {
        alert("Campo obrigatório.");
        document.getElementById("descarte").focus;
        return false;
    } else if (dataDiagnostico === "") {
        alert("Campo obrigatório.");
        document.getElementById("dataDiagnostico").focus;
        return false;
    } else if (horaDiagnostico === "") {
        alert("Campo obrigatório.");
        document.getElementById("horaDiagnostico").focus;
        return false;
    } else if (diagnostico === "") {
        alert("Campo obrigatório.");
        document.getElementById("diagnostico").focus;
        return false;
    } else if (sexo === "") {
        alert("Campo obrigatório.");
        document.getElementById("sexo").focus;
        return false;
    } else if (numDias === "") {
        alert("Campo obrigatório.");
        document.getElementById("numDias").focus;
        return false;
    } else if (reprodutor === "") {
        alert("Campo obrigatório.");
        document.getElementById("reprodutor").focus;
        return false;
    } else if (raca === "") {
        alert("Campo obrigatório.");
        document.getElementById("raca").focus;
        return false;
    } else if (dataCio === "") {
        alert("Campo obrigatório.");
        document.getElementById("dataCio").focus;
        return false;
    } else if (horaCio === "") {
        alert("Campo obrigatório.");
        document.getElementById("horaCio").focus;
        return false;
    } else if (cio === "") {
        alert("Campo obrigatório.");
        document.getElementById("cio").focus;
        return false;
    } else if (muco === "") {
        alert("Campo obrigatório.");
        document.getElementById("muco").focus;
        return false;
    } else if (te === "") {
        alert("Campo obrigatório.");
        document.getElementById("te").focus;
        return false;
    } else if (utero === "") {
        alert("Campo obrigatório.");
        document.getElementById("utero").focus;
        return false;
    } else if (ovD === "") {
        alert("Campo obrigatório.");
        document.getElementById("ovD").focus;
        return false;
    } else if (ovE === "") {
        alert("Campo obrigatório.");
        document.getElementById("ovE").focus;
        return false;
    } else if (base === "") {
        alert("Campo obrigatório.");
        document.getElementById("base").focus;
        return false;

    } else if (document.getElementById("acao").value === "gravarInfoInseminacao") {
        caminhourl = "/GeralViewController/Servlet?acao=gravarInfoInseminacao";
        form.action = caminhourl;
        form.submit();
    }
}

File: Geral-core/src/br/cefetmg/inf/geral/model/domain/RepDiagnostico.java
package br.cefetmg.inf.geral.model.domain;

import java.sql.Time;
import java.util.Date;

public class RepDiagnostico {
    private Long seq_ReproducaoDiagnostico;
    private Long numAnimal;
    private String nomAnimal;
    private String inseminadasCobertas;
    private String vazias;
    private String gestantes;
    private String emCrescimento;
    private String padrao;
    private String doadora;
    private String receptora;
    private String descarte;
    private Date dataDiagnostico;
    private Time horaDiagnostico;
    private String diagnostico;
    private String sexo;
    private String numDias;
    private String reprodutor;
    private String raca;
    private Date dataCio;
    private Time horaCio;
    private String cio;
    private String muco;
    private String te;
    private String utero;
    private String ovD;
    private String ovE;
    private String base;

    public Long getSeq_ReproducaoDiagnostico() {
        return seq_ReproducaoDiagnostico;
    }

    public void setSeq_ReproducaoDiagnostico(Long seq_ReproducaoDiagnostico) {
        this.seq_ReproducaoDiagnostico = seq_ReproducaoDiagnostico;
    }

    public Long getNumAnimal() {
        return numAnimal;
    }

    public void setNumAnimal(Long numAnimal) {
        this.numAnimal = numAnimal;
    }

    public String getNomAnimal() {
        return nomAnimal;
    }

    public void setNomAnimal(String nomAnimal) {
        this.nomAnimal = nomAnimal;
    }

    public String getInseminadasCobertas() {
        return inseminadasCobertas;
    }

    public void setInseminadasCobertas(String inseminadasCobertas) {
        this.inseminadasCobertas = inseminadasCobertas;
    }

    public String getVazias() {
        return vazias;
    }

    public void setVazias(String vazias) {
        this.vazias = vazias;
    }

    public String getGestantes() {
        return gestantes;
    }

    public void setGestantes(String gestantes) {
        this.gestantes = gestantes;
    }

    public String getEmCrescimento() {
        return emCrescimento;
    }

    public void setEmCrescimento(String emCrescimento) {
        this.emCrescimento = emCrescimento;
    }

    public String getPadrao() {
        return padrao;
    }

    public void setPadrao(String padrao) {
        this.padrao = padrao;
    }

    public String getDoadora() {
        return doadora;
    }

    public void setDoadora(String doadora) {
        this.doadora = doadora;
    }

    public String getReceptora() {
        return receptora;
    }

    public void setReceptora(String receptora) {
        this.receptora = receptora;
    }

    public String getDescarte() {
        return descarte;
    }

    public void setDescarte(String descarte) {
        this.descarte = descarte;
    }

    public Date getDataDiagnostico() {
        return dataDiagnostico;
    }

    public void setDataDiagnostico(Date dataDiagnostico) {
        this.dataDiagnostico = dataDiagnostico;
    }

    public Time getHoraDiagnostico() {
        return horaDiagnostico;
    }

    public void setHoraDiagnostico(Time horaDiagnostico) {
        this.horaDiagnostico = horaDiagnostico;
    }

    public String getDiagnostico() {
        return diagnostico;
    }

    public void setDiagnostico(String diagnostico) {
        this.diagnostico = diagnostico;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public String getNumDias() {
        return numDias;
    }

    public void setNumDias(String numDias) {
        this.numDias = numDias;
    }

    public String getReprodutor() {
        return reprodutor;
    }

    public void setReprodutor(String reprodutor) {
        this.reprodutor = reprodutor;
    }

    public String getRaca() {
        return raca;
    }

    public void setRaca(String raca) {
        this.raca = raca;
    }

    public Date getDataCio() {
        return dataCio;
    }

    public void setDataCio(Date dataCio) {
        this.dataCio = dataCio;
    }

    public Time getHoraCio() {
        return horaCio;
    }

    public void setHoraCio(Time horaCio) {
        this.horaCio = horaCio;
    }

    public String getCio() {
        return cio;
    }

    public void setCio(String cio) {
        this.cio = cio;
    }

    public String getMuco() {
        return muco;
    }

    public void setMuco(String muco) {
        this.muco = muco;
    }

    public String getTe() {
        return te;
    }

    public void setTe(String te) {
        this.te = te;
    }

    public String getUtero() {
        return utero;
    }

    public void setUtero(String utero) {
        this.utero = utero;
    }

    public String getOvD() {
        return ovD;
    }

    public void setOvD(String ovD) {
        this.ovD = ovD;
    }

    public String getOvE() {
        return ovE;
    }

    public void setOvE(String ovE) {
        this.ovE = ovE;
    }

    public String getBase() {
        return base;
    }

    public void setBase(String base) {
        this.base = base;
    }
}
